help/destinations/catalog/mobile-engagement/moengage.md

```diff
@@ -0,0 +1,141 @@
+
+See [Activate audience data to streaming segment export destinations](../../ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from [!DNL Adobe Experience Platform] to the [!DNL Moengage] destination, you need to go through the field mapping step.
+
+Mapping consists of creating a link between your [!DNL Experience Data Model] (XDM) schema fields in your [!DNL Platform] account, and their corresponding equivalents from the target destination.
+
+To correctly map your XDM fields to the [!DNL Braze] destination fields, follow these steps:
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Checkbox]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/segments.png)
+
+In the [!UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**.
+
+![Moengage Destination Add Mapping](../../assets/catalog/mobile-engagement/moengage/mapping.png)
+
+In the [!UICONTROL Source Field] section, click the arrow button next to the empty field.
+   
+![Moengage Destination Source Mapping](../../assets/catalog/mobile-engagement/moengage/mapping-source.png)
+
+In the [!UICONTROL Select source field] window, you can choose between two categories of XDM fields:
+* [!UICONTROL Select attributes]: use this option to map a specific field from your XDM schema to [!DNL Moengage] attribute.
+
+![Moengage Destination Mapping Source Attribute](../../assets/catalog/mobile-engagement/moengage/mapping-attributes.png)
+
+Choose your source field, then click **[!UICONTROL Select]**.
+
+In the [!UICONTROL Target Field] section, click the mapping icon to the right of the field.
+   
+![Moengage Destination Target Mapping](../../assets/catalog/mobile-engagement/moengage/mapping-target.png)
+
+In the [!UICONTROL Select target field] window, you can choose between two categories of target fields:
+* [!UICONTROL Select identity namespace]: Use this option to map [!DNL Platform] identity namespaces to [!DNL Braze] identity namespaces.
+* [!UICONTROL Select custom attributes]: Use this option to map XDM attributes to custom [!DNL Braze] attributes that you defined in your [!DNL Braze] account. <br> You can also use this option to rename existing XDM attributes into [!DNL Braze]. For instance, mapping a `lastName` XDM attribute to a custom `Last_Name` attribute in [!DNL Braze], will create the `Last_Name` attribute in [!DNL Braze], if it doesn't already exist, and map the `lastName` XDM attribute to it.
+  
+![Moengage Destination Target Mapping Fields](../../assets/catalog/mobile-engagement/moengage/mapping-target-fields.png)
+
+Choose your target field, then click **[!UICONTROL Select]**.
+
+You should now see your field mapping in the list.
+
+![Moengage Destination Mapping Complete](../../assets/catalog/mobile-engagement/moengage/mapping-complete.png)
```

Comment:
are the mappings in the screenshot realistic? for example, I see that we added user_id and anonymous_id in the same destination configuration. 

I also see that you mapped a city address attribute to a user ID identity. is that right?
similarly, you mapped a GAID identity to a postal code. is that right?

Reply:
just for an example i mapped like that. a single user can have user_id and anonymous_id both.

Reply:
attached more appropriate example